services/dynamic_addressbook_add_node.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.consensus">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+import "basic_types.proto";
+
+/**
+ * See [DynamicAddressBookService.addNode()](#proto.DynamicAddressBookService)
+ */
+message DynamicAddressBookAddNodeTransactionBody {
+    /**
+     * The unique entity id of the Node.
+     */
+    int64 nodeID = 1;
```

Comment:
Believe this field should be assigned by consensus nodes and not passed in. We should add this field to the receipt and consider if any other fields are necessary in receipt or TransactionRecord.

Reply:
removed this field.